app/src/main/res/layout/view_card_featured_article.xml

```diff
@@ -11,20 +11,26 @@
         android:layout_height="@dimen/view_card_header_height"
         app:layout_constraintTop_toTopOf="parent" />
 
-    <org.wikipedia.views.FaceAndColorDetectImageView
-        android:id="@+id/view_featured_article_card_image"
-        style="@style/SimpleDraweeViewPlaceholder"
+    <LinearLayout
```

Comment:
@sharvaniharan This can easily be a `FrameLayout`, which is a lighter component than `LinearLayout`. (I'll just update this PR)